src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
I've understood it that way: when node senses that it has not enough RAM, it notifies all local queues to start pushing data to disk. So suppose we have 2 nodes in cluster that are actively paging data to disk (there is no alarms yet), and first node runs out of free space. All nodes are alarmed at that point and publishers are blocked. But it is safe to continue paging on the node without disk alarm, so we can still relieve RAM pressure there. It needs to be done anyway, and there is no reason to wait for clearing alarms on other nodes.